### PR TITLE
Use options object for all configuration

### PR DESCRIPTION
**Pull request by HarryEMartland**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,19 @@ const registry = new Registry();
...
+
+```js
+const options = {};
+new PrometheusMetrics(options)
+```
+
+|Name                      |Description                                                             |Default       |
+|--------------------------|------------------------------------------------------------------------|--------------|
+|`circuits`                |A list or individual circuit breaker to create metrics for              |No circuits   |
+|`registry`                |An existing registry to use for prometheus metrics                      |A new registry|
```
Comment (lholmquist) [inline README.md:75]: i think the default here is undefined, or something like that

File: README.md [hunk 2]
```diff
@@ -58,6 +58,19 @@ const registry = new Registry();
 
 // create a circuit
 const circuit = new CircuitBreaker(functionThatMightFail);
-const metrics = new PrometheusMetrics(circuit, registry)
+const metrics = new PrometheusMetrics({ circuits: [circuit], registry: registry })
 ```
 
+## Options
+The second argument of the `PrometheusMetrics` constructor is an options object.
```
Comment (lholmquist) [inline README.md:65]: no longer the second argument

Comment (HarryEMartland) [reply README.md]: Thanks I have corrected this

File: README.md (hunk 1 above)
Comment (HarryEMartland) [reply README.md]: I have added some more information about this option

**Comment by HarryEMartland** [comment]
How does this look now? Should I close #18 and I can raise another MR once this is in master?

File: index.js [hunk 3]
```diff
@@ -26,23 +22,31 @@ class PrometheusMetrics {
       labelNames: ['name', 'event']
     });
 
-    this._summary = new this._client.Summary({
-      name: `circuit_perf`,
-      help: `A summary of all circuit's events`,
-      registers: [this._registry],
-      labelNames: ['name', 'event']
-    });
+    if (this.exposePerformanceMetrics()) {
```
Comment (lholmquist) [inline index.js:25]: I think this should be in the next PR instead of this one

File: index.js [hunk 4]
```diff
@@ -26,23 +22,31 @@ class PrometheusMetrics {
...
     }
 
-    if (circuits) {
-      this.add(circuits);
+    if (options.circuits) {
+      this.add(options.circuits);
     }
   }
 
+  exposePerformanceMetrics () {
```
Comment (lholmquist) [inline index.js:44]: same as the previous comment

File: index.js [hunk 5]
```diff
@@ -55,7 +59,8 @@ class PrometheusMetrics {
           this._counter.labels(circuit.name, eventName).inc();
         });
 
-        if (eventName === 'success' || eventName === 'failure') {
+        if (this.exposePerformanceMetrics() &&
```
Comment (lholmquist) [inline index.js:62]: same as the previous comment

**Comment by lholmquist** [comment]
@HarryEMartland thanks for keeping up with this :)

**Pull request merged**